src/vis/vis.js

```diff
@@ -403,19 +390,10 @@ var Vis = View.extend({
     // (eg: when the layer is hidden or it's SQL is changed)
     if (cartoDBLayerGroup) {
       var endpoint;
-      var mapsApiTemplate;
-      var userName;
-      var statTag;
-      var forceCors = true;
       var configGenerator;
 
       var datasource = data.datasource;
       if (datasource) {
```

Comment:
With this change isn't datasource always required? If so, perhaps should throw an error to indicate that it's missing if not set?